docs/index.rst

```diff
@@ -6,8 +6,223 @@
 The Zoo
 =======
 
+A smart service catalog providing an overview of your services' development and operations.
+
+Adding custom checks
+====================
+
+If you want to add your own checks to the zoo, you need to create your own project with a python package with two directories inside:
```

Comment:
Yep, here I forgot to add that the package provided must be placed in the directory specified by the `ZOO_AUDITING_ROOT ` environment variable, which is pointing to `zoo/auditing/standards`. Sorry, I just realized this...

Reply:
And do you think it's necessary to explain that `__init__.py` files are needed in the folders? That's implicit in the definition of package, but I'd like to know your opinion on this.

Reply:
I will include it, it is possible that someone not that familiar with python will try to implement it.

Reply:
Makes sense, thanks!